app/javascript/src/components/Common/Navbar/index.jsx

```diff
@@ -24,29 +23,50 @@ const NavBar = () => {
   };
 
   return (
-    <div className="bg-gray-100 nh-sidebar" key="sidebar">
-      <div className="nh-logo">
-        <div className="flex items-center justify-center w-8 h-8 rounded-md">
-          <i className="text-purple-500 ri-flashlight-fill ri-2x" />
-        </div>
-      </div>
-      <div className="flex flex-col items-center justify-between w-full h-full">
-        <div className="flex flex-col items-center justify-start w-full pt-4">
-          <NavItem title="Notes" link="/notes" icon="ri-file-text-line" />
-          <NavItem
-            title="Settings"
-            link="/settings"
-            icon="ri-settings-2-line"
-            subLinks={[
-              { title: "Change password", link: "/my/password/edit" },
-              { title: "My Profile", link: "/my/profile" }
-            ]}
-          />
-        </div>
-        <div className="mb-4">
-          <AccountDropdown handleLogout={handleLogout} />
-        </div>
-      </div>
+    <div className="flex flex-row items-start justify-start">
+      <Sidebar
+        appName="neetoUI"
+        collapsible
+        isCollapsed
+        navLinks={[
+          {
+            icon: function noRefCheck() {
+              return <Home />;
+            },
+            label: "Notes",
+            to: "/notes"
+          },
+          {
+            icon: function noRefCheck() {
+              return <UserCircle />;
+            },
+            label: "Contacts",
+            to: "/contacts"
+          },
+          {
+            icon: function noRefCheck() {
+              return <Settings />;
+            },
+            label: "Settings",
+            to: "/settings"
+          }
+        ]}
```

Comment:
Please move this to a `constants.js` file. Also, while exporting constants from a separate file, use the following convention - 
`export const NAV_LINKS`